File: robot/servo.py
import time
import random
import pyupm_servo as servo 

servo1 = servo.ES08A(5)#したのやつ
servo2 = servo.ES08A(6)
servo3 = servo.ES08A(7)
servo4 = servo.ES08A(8)
servo5 = servo.ES08A(9)#上のやつ。つまりここが手

# def setAngle(targetServo, angle):
#     targetServo.setAngle(angle)
#     print(targetServo,"move to angle : ",angle)



# Delete the servo object
del gServo 

File: robot/servo.js
var servoModule = require("jsupm_servo");

var servo1 = new servoModule.ES08A(3);
var servo2 = new servoModule.ES08A(5);
var servo3 = new servoModule.ES08A(6);
var servo4 = new servoModule.ES08A(9);
var servo5 = new servoModule.ES08A(11);

var timeOffSet = 2000;

function setServoAngle(servo,angle){
	servo.setAngle(angle);
	console.log("Set angle to " + angle);
	var nextAngle = Math.floor(Math.random()*90);
	setTimeout(function(){
		setServoAngle(servo,nextAngle);
	},timeOffSet);
}

console.log("start simasu get ready~")

setTimeout(function(){
		setServoAngle(servo1,90);
		setServoAngle(servo2,90);
		setServoAngle(servo3,90);
		setServoAngle(servo4,90);
		setServoAngle(servo5,0);

},1000);

process.on('SIGINT', function()
{
	console.log("Exiting...");
	process.exit(0);
});
